src/algebra/group/basic.lean

```diff
@@ -648,3 +649,107 @@ lemma commutator_element_def  {G : Type*} [group G] (g₁ g₂ : G) :
   ⁅g₁, g₂⁆ = g₁ * g₂ * g₁⁻¹ * g₂⁻¹ := rfl
 
 end commutator
+
+/-! ### Order dual -/
+
+open order_dual
+
+@[to_additive] instance [h : has_one α] : has_one αᵒᵈ := h
```

Comment:
Note that
```suggestion
@[to_additive] instance [has_one α] : has_one αᵒᵈ := ‹has_one α›
```
avoids `h :` appearing in the docs; but perhaps this is too minor to care about.

Reply:
Yeah, it's mostly harder to industrialise because of the line-wrapping 😦